scripts/geoid_tool.py

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env python
+"""
+Takes input reference dem data (point cloud or raster) and will transform from either geoid to ellipsoid, or ellipsoid to geoid.
+
+Usage:
+    geoid_tool.py <reference_data> [-e epsg] [-t transform_command] [-a asp_dir] [-d debug]
+
+Options:
+    -e epsg                   Desired EPSG code you want (for example, 32611)
+    -t transform_command      Either "to_geoid" or "to_ellipsoid"
+    -a asp_dir                Directory with ASP binary files
+    -d debug                  turns on debugging logging
+
+"""
+import json
+from docopt import docopt
+from glob import glob
+from os.path import abspath,exists, join, basename, dirname
+import rasterio
+import laspy
+from pyproj import CRS
+from datetime import datetime
+import logging
+import sys
+import os
+
+# local imports
+from dir_space_strip import replace_white_spaces
+from laz2dem import cl_call
+
+
+
+
+def check_datum_do_not_match(datum, datum_ref): 
+    '''
+    Ensure vertical datums do not already match. This is performed after `code_datum()`.
+
+    Inputs:
+        datum -  desired vertical datum input by user.
+        datum_ref - vertical datum read from input reference data.
+
+    Outputs:
+        None
+
+    '''
+
+    if 'WGS'==datum_ref and 'WGS'==datum :
```

Comment:
MINOR style suggestion:
Since we are using the strings for `WGS` and `NAD` a lot in this code, I would personally declare a constant at the top of this script (~line 31) and then re-use throughout